person_class.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    if of_age? || @parent_permission
+      true
+    else
+      false
+    end
+  end
```

Comment:
- [ ] In order to simpily your code base, the `can_use_services?` method is a Ruby method that returns a boolean value, the **`?`** at the end of the method name implies that it's a predicate method that either return `true or false`. ℹ️ 

```suggestion
 def can_use_services? 
    of_age?() || @parent_permission
  end
```